spec/support/fake_heroku.rb

```diff
@@ -6,8 +6,9 @@ def initialize(args)
   end
 
   def run!
-    if @args.first == "plugins"
-      puts "heroku-pipelines@0.29.0"
+    if @args.first == "help"
+      # spaces in front are intentional
+      puts "  pipelines      #  manage collections of apps in pipelines"
```

Comment:
In general I'm not a fan of comments, so if there's a way to remove them without losing information it would be great. Let's see what @tute thinks.

This is a small thing, so I think we can merge either way.